Dockerfile

```diff
@@ -16,15 +17,17 @@ ENV CGO_ENABLED=0
 ENV GOOS=linux
 ENV GOARCH=amd64
 ENV GO111MODULE=on
-RUN go build -o /hauser \
+RUN go build -o /bin/hauser \
     -ldflags "-w -extldflags \"-static\" -X \"main.version=$(cat VERSION)\"" \
     .
 
 FROM scratch
-WORKDIR /
 COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 COPY --from=builder /etc/passwd /etc/passwd
-COPY --from=builder /hauser /bin/hauser
+COPY --from=builder /etc/group /etc/group
+COPY --from=builder /bin/hauser /bin/hauser
+COPY --from=builder --chown=hauser /hauser /hauser
+WORKDIR /hauser
```

Comment:
Perhaps a README update is warranted, to indicate that the config file used should omit the `TmpDir` param or, if present, it must be relative or start with `/hauser/`?

Is there any value in a container _preserving_ these files from run to run? Does hauser need to save a "cursor" to disk, so it can correctly resume after a restart? If so, then doing so just in the temporary volume won't be good since it means it would try to re-download all history if the container is restarted. (Maybe that cursor is saved somewhere else, like BQ?)

If it does need to be saved to the local file-system, then we'd also need directions about mounting that as a volume to the container, so it gets persisted in the host's file system. And to do that with a non-root user, we may want to pick a hard-coded UID and GID for the user/group. That way, users can do `chown :1024 ~/hauser-cursor` to make sure the container can write to their host file system (where "1024" in this example is the UID of the "hauser" user inside the container). (For more context: https://medium.com/@nielssj/docker-volumes-and-file-system-permissions-772c1aee23ca).

Reply:
> Perhaps a README update is warranted, to indicate that the config file used should omit the TmpDir param or, if present, it must be relative or start with /hauser/?

I think this is necessary regardless. I can make an update with this PR to include some guidance for running the hauser image.

> Is there any value in a container preserving these files from run to run? Does hauser need to save a "cursor" to disk, so it can correctly resume after a restart? If so, then doing so just in the temporary volume won't be good since it means it would try to re-download all history if the container is restarted. (Maybe that cursor is saved somewhere else, like BQ?)

In short, no. If the process is running in "local" mode, then it will save the cursor to wherever the files are being stored, which should be in a mounted folder anyways. For all other processes, the cursor is stored in the destination (a dedicated table for database destinations and a cursor file for storage destinations).